### PR TITLE
flix 0.27.0 (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

File: Formula/flix.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    (bin/"flix").write <<~EOS
+      #!/bin/bash
+      CLASSPATH="#{prefix}/flix-#{version}.jar:." exec "#{Formula["openjdk"].opt_bin}/java" -jar #{prefix}/flix-#{version}.jar "$@"
+    EOS
+  end
+
+  test do
+    system bin/"flix", "init"
+    system bin/"flix", "run"
+    system bin/"flix", "test"
```
Comment (chenrui333) [inline Formula/flix.rb:25]: anyway to assert the response or artifact?

File: Formula/flix.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  head "https://github.com/flix/flix.git", branch: "master"
+
+  depends_on "gradle" => :build
+  depends_on "scala" => :build
+  depends_on "openjdk"
+
+  def install
+    system Formula["gradle"].bin/"gradle", "build", "jar"
+    prefix.install "build/libs/flix-#{version}.jar"
+    (bin/"flix").write <<~EOS
```
Comment (SMillerDev) [inline Formula/flix.rb:16]: Does this not work with `write_jar_script`?

**Comment by chenrui333** [comment]
ping @pthariensflame

**Comment by pthariensflame** [comment]
Sorry for the wait!  I'll get on this again this afternoon.  😊

File: Formula/flix.rb (hunk 1 above)
Comment (pthariensflame) [reply Formula/flix.rb:25]: It prints "Hello World" but doesn't *only* do that, and I wasn't sure how to separate out the parts there.

File: Formula/flix.rb (hunk 2 above)
Comment (pthariensflame) [reply Formula/flix.rb:16]: I didn't know that existed!  I copied this part from the `antlr` formula.

File: Formula/flix.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/flix.rb:22]:
```suggestion
    system bin/"flix", "init"
    system bin/"flix", "run"
    assert_match "Hello World", shell_output(bin/"flix test")
```

**Review by SMillerDev — COMMENTED** [review]
LGTM, but with the test suggestion it would be even better

File: Formula/flix.rb (hunk 1 above)
Comment (pthariensflame) [reply Formula/flix.rb:22]: Thank you!  ❤️

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @pthariensflame!!

**Pull request closed without merging**